src/main/java/randoop/reflection/SignatureParser.java

```diff
@@ -0,0 +1,154 @@
+package randoop.reflection;
+
+import java.lang.reflect.AccessibleObject;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/** Parses type signature strings used to identify methods and constructors in input. */
+// TODO: This duplicates should be factored into a separate source set (aka, module) so that it
```

Comment:
How much work would it be to do this?
Because I made improvements to this section, they are already out of synch with one another.